testing/file_test/test_file.cpp

```diff
@@ -93,9 +96,53 @@ struct SplitState {
   int file_index = 0;
 };
 
+static auto ExtractFilePathFromUri(llvm::StringRef uri)
+    -> std::optional<llvm::StringRef> {
+  static constexpr llvm::StringRef FilePrefix = "file:/";
+  if (!uri.starts_with(FilePrefix)) {
+    return std::nullopt;
+  }
+  return uri.drop_front(FilePrefix.size());
+}
+
+static auto AutoFillDidOpenParams(llvm::json::Object* params,
+                                  llvm::ArrayRef<TestFile::Split> splits)
+    -> ErrorOr<Success> {
+  auto* text_document = params->getObject("textDocument");
+  if (text_document == nullptr) {
+    return Success{};
```

Comment:
```suggestion
    return Success();
```

Style: for init, we avoid braces when possible. (https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/cpp_style_guide.md , "For initialization")

Reply:
Fixed.